core/server/master/src/main/java/alluxio/master/file/meta/MutableInodeDirectory.java

```diff
@@ -111,6 +111,8 @@ public MutableInodeDirectory setDefaultACL(DefaultAccessControlList acl) {
   /**
    * Generates client file info for a folder.
    *
+   * xAttr is intentionally left out as the information is not yet exposed to clients
```

Comment:
Would it make sense to support the read path of the xattrs, meaning, we expose the map in the file info. We can always add the "write" portion of the API at a later time. This way, if a ufs wants to set values in xattrs, clients/cli can still view the values.

Reply:
For the time being, if we don't plan to expose the extended attributes to users to write, I don't think they should be able to read them. I think it keeps the API cleaner and less confusing. If clients can't set them, why do they need to be read? If you have a strong use case for clients reading UFS attributes now I could be convinced to add it.